src/Jobs/PtpJob.php

```diff
@@ -0,0 +1,236 @@
+<?php
+namespace Biigle\Modules\Ptp\Jobs;
+use Biigle\Jobs\Job as BaseJob;
+use Biigle\User;
+use Biigle\Image;
+use Biigle\ImageAnnotation;
+use Biigle\Modules\Ptp\PtpExpectedArea;
+use Exception;
+use File;
+use FileCache;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Bus\Batchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Storage;
+
+class PtpJob extends BaseJob implements ShouldQueue
```

Comment:
This class has several attributes that were copy/pasted but are not relevant for this job.